packages/blade/src/utils/storybook/Sandbox/baseCode.ts

```diff
@@ -0,0 +1,293 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import dedent from 'dedent';
+// @ts-expect-error We don't resolve JSON files right now. didn't want to change TS config for single JSON
+import packageJson from '../../../../package.json'; // eslint-disable-line
+
+const getBladeVersion = (): string => {
+  // We don't publish codesandbox ci on master so version is not present
+  const isMaster = process.env.GITHUB_REF === 'refs/heads/master';
+  const sha = process.env.GITHUB_SHA;
+  if (sha && !isMaster) {
+    const shortSha = sha.slice(0, 8);
+    return `https://pkg.csb.dev/razorpay/blade/commit/${shortSha}/@razorpay/blade`;
```

Comment:
this means it shall run on PRs right? but it hasn;t beeing doesn't running on our PRs as well even the sandpack CI? 🤔

Reply:
It just uses version published by codesandbox ci in stackblitz. Stackblitz in itself doesn't run on PRs 🤔